hm_9/hm_09.py

```diff
@@ -0,0 +1,64 @@
+term = int(input("Enter a term here: "))
+
+# task_number_one
+
+
+def fib(n):
+    a, b = 0, 1
+    for item in range(n + 1):
+        yield a
+        a, b = b, a + b
+
+
+fib_generator = fib(term)
+for item in fib_generator:
```

Comment:
it is not required to use for in here. You can transform your generator to a list and get its last value:
```
result = list(fib_generator)[-1]
print(result)